### ISSUE TITLE
Sleep when no checkboxes have been found

### ISSUE THREAD

**Issue by demiantres** [mrishab/google-photos-delete-tool#20]
Maybe sleep some time when no checkboxes have been found so that the page can properly load:

```

    do {
        checkboxes = document.querySelectorAll(ELEMENT_SELECTORS['checkboxClass']);
		if (checkboxes.length <= 0)
			Sleep(2000); 
    } while (checkboxes.length <= 0 && attemptCount++ < MAX_RETRIES);

```

**Comment by CyberFlexie** [comment]
I have used this and seems to work! thanks!

**Comment by shtse8** [comment]
Agreed — the original loop waits on a fixed retry counter with no pause. The maintained continuation polls with real delays (waitUntil + exponential backoff), so slow page loads don't cause false "no more images" exits: https://github.com/shtse8/Google-Photos-Delete-Tool (Chrome extension, userscript, bookmarklet, or console paste).
